src/Content.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Graby;
+
+/**
+ * Immutable container for Graby result.
+ */
+final class Content
+{
+    private int $status;
```

Comment:
Maybe instead of HTTP properties, we could just store PSR-7 response object.

Reply:
As far as I remember we've added the `status` to be able to retrieve it in wallabag (same as for `headers`).
So maybe we can ditch both and keep one `response`?

Reply:
Yes, though I would want to return that with `Graby\Extractor\HttpClient::fetch()`. And we would still need some mechanism to pass the effective URL. I guess we can decide on that later.